connector/oidc/oidc_test.go

```diff
@@ -412,6 +415,178 @@ func TestRefresh(t *testing.T) {
+	assertParamValue(t, values, "state", "1234")
+	assertParamValue(t, values, "response_type", "code")
+	assertParamValue(t, values, "scope", "openid profile email")
+}
+
+func TestLoginURLClientIdError(t *testing.T) {
+	cfg := Config{
+		ClientID:    "client",
+		RedirectURI: "callback",
+		AdditionalAuthRequestParams: map[string]string{
+			"client_id": "not-so-fast",
+		},
+	}
+	_, err := testLoginURL(t, cfg, "1234")
+	assert.EqualErrorf(t, err, "parameter 'client_id' is already managed by this connector", "")
+}
+
+func TestLoginURLRedirectURIError(t *testing.T) {
+	cfg := Config{
+		ClientID:    "client",
+		RedirectURI: "callback",
+		AdditionalAuthRequestParams: map[string]string{
+			"redirect_uri": "not-so-fast",
+		},
+	}
+	_, err := testLoginURL(t, cfg, "1234")
+	assert.EqualErrorf(t, err, "parameter 'redirect_uri' is already managed by this connector", "")
+}
+
+func TestLoginURLStateError(t *testing.T) {
+	cfg := Config{
+		ClientID:    "client",
+		RedirectURI: "callback",
+		AdditionalAuthRequestParams: map[string]string{
+			"state": "not-so-fast",
+		},
+	}
+	_, err := testLoginURL(t, cfg, "1234")
+	assert.EqualErrorf(t, err, "parameter 'state' is already managed by this connector", "")
+}
+
+func TestLoginURLHostedDomainsError(t *testing.T) {
+	cfg := Config{
+		ClientID:    "client",
+		RedirectURI: "callback",
+		AdditionalAuthRequestParams: map[string]string{
+			"hd": "not-so-fast",
+		},
+	}
+	_, err := testLoginURL(t, cfg, "1234")
+	assert.EqualErrorf(t, err, "parameter 'hd' is already managed by this connector", "")
+}
+
+func TestLoginURLResponseTypeError(t *testing.T) {
+	cfg := Config{
+		ClientID:    "client",
+		RedirectURI: "callback",
+		AdditionalAuthRequestParams: map[string]string{
+			"response_type": "not-so-fast",
+		},
+	}
+	_, err := testLoginURL(t, cfg, "1234")
+	assert.EqualErrorf(t, err, "parameter 'response_type' is already managed by this connector", "")
+}
+
+func TestLoginURLScopeTypeError(t *testing.T) {
+	cfg := Config{
+		ClientID:    "client",
+		RedirectURI: "callback",
+		AdditionalAuthRequestParams: map[string]string{
+			"scope": "not-so-fast",
+		},
+	}
+	_, err := testLoginURL(t, cfg, "1234")
+	assert.EqualErrorf(t, err, "parameter 'scope' is already managed by this connector", "")
+}
+
+func TestLoginURLPromptError(t *testing.T) {
+	cfg := Config{
+		ClientID:    "client",
+		RedirectURI: "callback",
+		AdditionalAuthRequestParams: map[string]string{
+			"prompt": "not-so-fast",
+		},
+	}
+	_, err := testLoginURL(t, cfg, "1234")
+	assert.EqualErrorf(t, err, "parameter 'prompt' is already managed by this connector", "")
+}
+
+func TestLoginURLAcrValuesError(t *testing.T) {
+	cfg := Config{
+		ClientID:    "client",
+		RedirectURI: "callback",
+		AdditionalAuthRequestParams: map[string]string{
+			"acr_values": "not-so-fast",
+		},
+	}
+	_, err := testLoginURL(t, cfg, "1234")
+	assert.EqualErrorf(t, err, "parameter 'acr_values' is already managed by this connector", "")
+}
```

Comment:
Would you mind merging these into a single test function using table-driven tests?